crates/ruma-events/src/enums.rs

```diff
@@ -100,8 +100,9 @@ event_enum! {
         #[ruma_enum(alias = "m.voice")]
         "org.matrix.msc3245.voice.v2" => super::voice,
         #[cfg(feature = "unstable-msc4075")]
+        #[cfg(feature = "unstable-msc3401")]
```

Comment:
Why are there 2 feature flags for this variant? If `unstable-msc4075` requires `unstable-msc3401`, it should be handled in the Cargo manifest of the crate.